### PR TITLE
Remove isBEBEnabled flag

### PR DESCRIPTION
**Pull request by marcobebway**
**Description**

Remove isBEBEnabled flag.

### CODE REVIEW

**Review by alexandra-simeonova — APPROVED** [review]

File: components/application-connectivity-validator/README.md [hunk 1]
```diff
@@ -3,19 +3,19 @@
...
 - **tenant** is the tenant of the Application for which the proxy is deployed. Omitted if empty.
 - **group** is the group of the Application for which the proxy is deployed. Omitted if empty.
-- **eventServicePathPrefixV1** is the path prefix for which requests are forwarded to the Event Service V1 API. The default value is `/v1/events`.
-- **eventServicePathPrefixV2** is the path prefix for which requests are forwarded to the Event Service V2 API. The default value is `/v2/events`.
-- **eventServiceHost** is the host and the port of the Event Service. The default value is `events-api:8080`.
+- **eventingPathPrefixV1** is the path prefix for which requests are forwarded to the Eventing Publisher V1 API. The default value is `/v1/events`.
+- **eventingPathPrefixV2** is the path prefix for which requests are forwarded to the Eventing Publisher V2 API. The default value is `/v2/events`.
+- **eventingPublisherHost** is the host and the port of the Eventing Publisher. The default value is `events-api:8080`.
 - **eventMeshDestinationPath** is the destination path for the requests coming to the Event Mesh. The default value is `/`.
```
[the remark this answers is not in the document]
Comment (marcobebway) [reply components/application-connectivity-validator/README.md]: Done

File: components/application-connectivity-validator/cmd/applicationconnectivityvalidator/applicationconnectivityvalidator.go [hunk 2]
```diff
@@ -61,11 +61,10 @@ func main() {
...
 		options.tenant,
-		options.eventServicePathPrefixV1,
-		options.eventServicePathPrefixV2,
-		options.eventServiceHost,
+		options.eventingPathPrefixV1,
+		options.eventingPathPrefixV2,
+		options.eventingPublisherHost,
 		options.eventMeshPathPrefix,
-		options.eventMeshHost,
 		options.eventMeshDestinationPath,
```
[the remark this answers is not in the document]
Comment (marcobebway) [reply components/application-connectivity-validator/cmd/applicationconnectivityvalidator/applicationconnectivityvalidator.go]: Done

File: components/application-connectivity-validator/cmd/applicationconnectivityvalidator/options.go [hunk 3]
```diff
@@ -44,12 +43,11 @@ func parseOptions() (*options, error) {
...
 	group := flag.String("group", "", "Name of the application group")
-	eventServicePathPrefixV1 := flag.String("eventServicePathPrefixV1", "/v1/events", "Prefix of paths that will be directed to the Event Service V1")
-	eventServicePathPrefixV2 := flag.String("eventServicePathPrefixV2", "/v2/events", "Prefix of paths that will be directed to the Event Service V2")
-	eventServiceHost := flag.String("eventServiceHost", "events-api:8080", "Host (and port) of the Event Service")
+	eventingPathPrefixV1 := flag.String("eventingPathPrefixV1", "/v1/events", "Prefix of paths that will be directed to Kyma Eventing V1")
+	eventingPathPrefixV2 := flag.String("eventingPathPrefixV2", "/v2/events", "Prefix of paths that will be directed to Kyma Eventing V2")
+	eventingPublisherHost := flag.String("eventingPublisherHost", "eventing-event-publisher-proxy.kyma-system", "Host (and port) of the Eventing Publisher")
+	eventMeshDestinationPath := flag.String("eventMeshDestinationPath", "/publish", "Path of the destination of the requests to the Event Mesh")
 	eventMeshPathPrefix := flag.String("eventMeshPathPrefix", "/events", "Prefix of paths that will be directed to the Event Mesh")
```
[the remark this answers is not in the document]
Comment (marcobebway) [reply components/application-connectivity-validator/cmd/applicationconnectivityvalidator/options.go]: Done

File: components/application-connectivity-validator/internal/validationproxy/handler.go [hunk 4]
```diff
@@ -41,20 +39,18 @@ type Cache interface {
...
-
-	eventsProxy      *httputil.ReverseProxy
-	eventMeshProxy   *httputil.ReverseProxy
-	appRegistryProxy *httputil.ReverseProxy
+	group                 string
+	tenant                string
+	eventingPathPrefixV1  string
+	eventingPathPrefixV2  string
+	eventingPublisherHost string
+	eventMeshPathPrefix   string
```
[the remark this answers is not in the document]
Comment (marcobebway) [reply components/application-connectivity-validator/internal/validationproxy/handler.go]: Renamed to `eventingPathPrefixEvents`.

File: components/application-connectivity-validator/internal/validationproxy/handler_test.go [hunk 5]
```diff
@@ -290,44 +284,11 @@ func TestProxyHandler_ProxyAppConnectorRequests(t *testing.T) {
...
-				recorder := httptest.NewRecorder()
-
-				// when
-				proxyHandler.ProxyAppConnectorRequests(recorder, req)
-
-				// then
-				assert.Equal(t, testCase.expectedStatus, recorder.Code)
-			})
-
 			t.Run("should proxy event mesh request when "+testCase.caseDescription, func(t *testing.T) {
```
[the remark this answers is not in the document]
Comment (marcobebway) [reply components/application-connectivity-validator/internal/validationproxy/handler_test.go]: Done

File: components/application-connectivity-validator/cmd/applicationconnectivityvalidator/options.go [hunk 6]
```diff
@@ -44,12 +43,11 @@ func parseOptions() (*options, error) {
...
 	tenant := flag.String("tenant", "", "Name of the application tenant")
 	group := flag.String("group", "", "Name of the application group")
-	eventServicePathPrefixV1 := flag.String("eventServicePathPrefixV1", "/v1/events", "Prefix of paths that will be directed to the Event Service V1")
-	eventServicePathPrefixV2 := flag.String("eventServicePathPrefixV2", "/v2/events", "Prefix of paths that will be directed to the Event Service V2")
-	eventServiceHost := flag.String("eventServiceHost", "events-api:8080", "Host (and port) of the Event Service")
+	eventingPathPrefixV1 := flag.String("eventingPathPrefixV1", "/v1/events", "Prefix of paths that will be directed to Kyma Eventing V1")
+	eventingPathPrefixV2 := flag.String("eventingPathPrefixV2", "/v2/events", "Prefix of paths that will be directed to Kyma Eventing V2")
+	eventingPublisherHost := flag.String("eventingPublisherHost", "eventing-event-publisher-proxy.kyma-system", "Host (and port) of the Eventing Publisher")
+	eventMeshDestinationPath := flag.String("eventMeshDestinationPath", "/publish", "Path of the destination of the requests to the Event Mesh")
```
[the remark this answers is not in the document]
Comment (marcobebway) [reply components/application-connectivity-validator/cmd/applicationconnectivityvalidator/options.go]: Done

File: components/application-connectivity-validator/cmd/applicationconnectivityvalidator/applicationconnectivityvalidator.go [hunk 7]
```diff
@@ -61,11 +61,10 @@ func main() {
 	proxyHandler := validationproxy.NewProxyHandler(
 		options.group,
 		options.tenant,
-		options.eventServicePathPrefixV1,
-		options.eventServicePathPrefixV2,
-		options.eventServiceHost,
+		options.eventingPathPrefixV1,
+		options.eventingPathPrefixV2,
+		options.eventingPublisherHost,
 		options.eventMeshPathPrefix,
```
[the remark this answers is not in the document]
Comment (marcobebway) [reply components/application-connectivity-validator/cmd/applicationconnectivityvalidator/applicationconnectivityvalidator.go]: Renamed to `eventingPathPrefixEvents`.

**Review by sayanh — APPROVED** [review]

**Review by alexandra-simeonova — APPROVED** [review]

**Review by crabtree — APPROVED** [review]

**Pull request merged**